content/docs/glossary/claimable-balance.mdx

```diff
@@ -158,9 +239,31 @@ if results, ok := txResult.OperationResults(); ok {
 balances, err := client.ClaimableBalances(sdk.ClaimableBalanceRequest{Claimant: B})
 check(err)
 balanceId := balances.Embedded.Records[0].BalanceID
+```
 
-// Now Account B can actually claim the balance, provided too much time
-// hasn't elapsed, or Account A can "reclaim" it otherwise.
+</CodeExample>
+
+With the claimable balance ID acquired, either Account B or A can actually submit a claim, depending on which predicate is fulfilled. We'll assume here that a minute has passed, so Account A just reclaims the balance entry.
+
+<CodeExample>
+
+```js
+let claimBalance = sdk.Operation.claimClaimableBalance({ balanceId: balanceId });
+console.log(A.publicKey(), "claiming", balanceId);
+
+tx = new sdk.TransactionBuilder(aAccount, {fee: sdk.BASE_FEE})
+  .addOperation(claimBalance)
+  .setNetworkPassphrase(sdk.Networks.TESTNET)
+  .setTimeout(180)
+  .build();
+
+tx.sign(A);
+return await server.submitTransaction(tx).catch(function (err) {
+  console.error(`Tx submission failed: ${err}`)
+});
```

Comment:
This code is not inside any specific function in this example so you shouldn't be returning anything. An `await` without `return` is the way to go:

```suggestion
await server.submitTransaction(tx).catch(function (err) {
  console.error(`Tx submission failed: ${err}`)
});
```

Reply:
Yep, excellent point. I think this is a copy-paste remnant from when I was actually *running* this code, before splitting it up into this "tutorial format".